src/ISADotnet/API/Comment.fs

```diff
@@ -1 +1,40 @@
-namespace ISADotNet.API
+namespace ISADotNet
```

Comment:
Was this namespace change made by heart?

Reply:
Wait this `fs` is supposed to be in the `datamodel `folder

Reply:
Yeah this slip up propably happened during some merge conflicts, good catch! Changed it.